tests/test_pipeline.py

```diff
@@ -18,7 +21,11 @@ def _test(files: List, config_path: str) -> None:
     ] + [
         path.join(tricky_path, file) for file in files[-num_regular:]
     ]
-    subprocess.check_call([build, config_path] + input_files_paths)
+    try:
+        subprocess.check_call([build, config_path] + input_files_paths)
+    except subprocess.CalledProcessError:
+        pytest.fail(f"Running gorshochek with config \"{config_path}\" " +
+                    f"on files [{', '.join(input_files_paths)}] ended with non-zero code")
```

Comment:
This test, so if something happened unexpectedly you should print here. In this step, we check that transformation doesn't break the code.
If the subprocess raises this exception test should print information about the current transformation, some words about what happened. Also, it would be great to print clang output, so we can see what is wrong with a transformed code.